pants.toml

```diff
@@ -2,7 +2,35 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 [GLOBAL]
-pants_version = "2.0.0.dev0"
-v1 = false
-backend_packages = []
-backend_packages2 = []
+pants_version = "2.0.0.dev1"
+
+pythonpath = ["pants-plugins"]
```

Comment:
Might be worth elaborating that this is so that we can activate the plugin from the repo sources, as opposed to from a published artifact.